app/models/course_participant.rb

```diff
@@ -27,6 +27,9 @@ def self.import(row_hash, _row_header = nil, session, id)
     unless CourseParticipant.exists?(user_id: user.id, parent_id: id)
       CourseParticipant.create(user_id: user.id, parent_id: id)
     end
+    password = "password"#user.password
+    prepared_mail = MailerHelper.send_mail_to_user(user, "Your Expertiza account and password have been created.", "user_welcome", password)
```

Comment:
This duplicates code from above.  Make it a call to another method, which is implemented in one place.